ui/app/AppLayouts/Wallet/controls/ManageTokensGroupDelegate.qml

```diff
@@ -97,25 +94,25 @@ DropArea {
                     font.weight: Font.Medium
                 }
 
-                StatusBaseText {
-                    Layout.leftMargin: -parent.spacing/2
-                    text: "• %1".arg(root.isCollectible ? qsTr("%n collectible(s)", "", root.childCount) : qsTr("%n asset(s)", "", root.childCount))
-                    elide: Text.ElideRight
-                    color: Theme.palette.baseColor1
-                    maximumLineCount: 1
-                    visible: !root.communityGroupsExpanded
-                }
-
                 Item { Layout.fillWidth: true }
 
+                ManageTokensCommunityTag {
```

Comment:
![Screenshot from 2024-01-17 11-50-17](https://github.com/status-im/status-desktop/assets/20650004/5de61c47-f69b-4285-a207-9eb1394e66d3)
Not sure what's the limit for community name but the tag width should be limited, e.g. to the half of available width and elided when longer.